.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject
```

Comment:
Hey, just add it to Your global gitignore and you will not need to ignore it anymore in any other repository. Read more here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore